digital_benchmark/instagramBenchmark/templates/auth.html

```diff
@@ -0,0 +1,16 @@
+{% load static%}
+<link rel='stylesheet' type='text/css' href="{% static '/style.css' %}"/>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+<script>
+    function instaLogin() {
+            url="https://api.instagram.com/oauth/authorize/?client_id=4d8f538893ba481f88c0614865dc9310&redirect_uri=http://127.0.0.1:8000/instagramBenchmark/instaLogin&response_type=token"
```

Comment:
Use a Django Form to do this.

Reply:
This link is taking user to login screen provided by Instagram API, we are just using a single button to go to that link. Do we still need a django form??